src/styles/components/header.scss

```diff
@@ -1,2 +1,78 @@
 .header {
+    background: var(--color-background-darker);
+    font-size: 2rem;
+    font-weight: 600;
+    border-bottom: 1px solid #5d5c61;
+    
+
+    &__content{
+        // height: 50px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    &__content-item{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-shrink: 5;
+        margin-left: 40px;
+        padding: 15px 0;
+    }
+
+    &__content-item:nth-of-type(2){
+        margin-right: 40px;
+        margin-left: 0px;
+    }
+
+    &__menu-item{
+        display: inline-block;
+        margin-left: 45px;
+        cursor: pointer;
+        text-decoration: none;
+        transform: translateY(1.3px);
+
+        &:visited{
+            color: white;
+        }
+
+        &:first-child{
+            margin-left: 0;
+        }
+
+        &:last-child{
```

Comment:
better to have this on hover, or when the link is active , remove it on off last child for now

Reply:
I added hover to the last child. I like the idea of it being purple after being clicked too. I have added the hover and left the visited state out.